samples/system-test/buckets.test.js

```diff
@@ -141,3 +141,47 @@ it('should delete a bucket', async () => {
   const [exists] = await bucket.exists();
   assert.strictEqual(exists, false);
 });
+
+it('should enforce public access prevention on a bucket', async () => {
```

Comment:
Debating whether this should test that public access prevention is actually enforced (ie that you can't make data public when the feature is turned on). Maybe this belongs in a library integration test though.

Reply:
Discussed with @tritone and determined that this logic will exist in a library system test